README.md

```diff
@@ -6,23 +6,49 @@ https://internado.azurewebsites.net/
 
 ## :computer: Running Locally
 
-### Running server
+### Running Job/Auth Servers
+| Server         | IP        | PORT | Example                   |
+|:----------------:|:-----------:|:------:|:---------------------------:|
+| Job Search     | localhost | 5000 | http://localhost:5000/api |
+| Authentication | localhost | 5050 | http://localhost:5050/api |
+
 Install all necessary node files and places them in a node_modules root directory folder
 ```console
 foo@bar:~$ npm install
 ```
-Builds the project and places the necesary files in a dist root directory folder
+Builds, Lints and Runs the corresponding server
+```console
+foo@bar:~$ npm start
+```
+#### Useful scripts
+* clean : Clears dist folder (where ts files are transpiled into js)
+```console
+foo@bar:~$ npm run clean
+```
+
+* lint : Runs ts linter
+```console
+foo@bar:~$ npm run lint
+```
+
+* build : Builds ts files (found in src folder) into js (to be generated into dist folder) 
 ```console
 foo@bar:~$ npm run build
 ```
-Runs the server (currently on http://localhost:5000/)
+
+* dev : Auto-builds ts files into js and runs server
```

Comment:
OOh I so it auto-rebuilds on change like react. That's great.

Reply:
Actually it only auto builds for the auth/job servers but I can probably hook something similar for the client

Reply:
Oh no I understood that. I was just saying that the client already does this.